### ISSUE TITLE
[HL2DM] physics map/server bug

### ISSUE THREAD

**Issue by adelina1985a** [ValveSoftware/Source-1-Games#1647]
https://www.youtube.com/watch?v=ZMcIvUCdTQg
So far I've seen the bug only on kbx servers/maps and thought it was a prop bug on the custom kbh map: http://cloud-3.steampowered.com/ugc/684841102034987006/157BD7B88D2510670BCAC104EA59733C91F3AE89/
Turns out, it's regardless of server and of map, or maybe it starts on one map but stays even after map change, manifesting itself even on stock maps like dm_lockdown.

I think it's also in CSS but I haven't played it to see it and verify if it's an engine bug. I only read a few examples like this one here: http://www.mr-community.com/forums/index.php?/topic/4784-glitchy-props/

Basically they fall through the floor, and other walls, railings etc, float around til reaching one edge of the map.
The only known fix is a server restart.

**Comment by Xeogin** [comment]
I recall this happening on dm_democracy, in particularly if you jumped on the barrels before anything touched them. I don't believe this occurs anymore though.

**Comment by adelina1985a** [comment]
I have recently experienced this with a cart in dm_overwatch, on an updated server.
